.github/workflows/lint.yml

```diff
@@ -37,24 +37,24 @@ jobs:
         with:
           node-version: 18
 
-      - name: Install Node.js dependencies
-        run: npm ci
+      - name: Install Cypress
+        run: npm i cypress
```

Comment:
is there a reason why this needs its own step? is it for the `run cypress tests` action down below?

Reply:
It is. Would it be better to integrate these two into one step?

Reply:
Have corrected this